src/icons/emojis/svgs/AnxiousEmoji.tsx

```diff
@@ -10,10 +11,11 @@ function SvgAnxiousEmoji(props: React.SVGProps<SVGSVGElement>) {
       xmlns="http://www.w3.org/2000/svg"
       {...props}
     >
+      <title>Anxious Emoji</title>
       <path
         fillRule="evenodd"
         clipRule="evenodd"
-        d="M28.086 3l-2.267 2.809.972.785 2.268-2.809L28.086 3zm-8.643.789l.973-.785 2.266 2.81-.973.785-2.266-2.81zM12.786 19.04l3.552-2.204.659 1.062-3.552 2.204-.66-1.063zm17.47-1.14l.66-1.062 3.552 2.205-.66 1.062-3.551-2.205zm1.104 6.235a1.18 1.18 0 100-2.36 1.18 1.18 0 000 2.36zm-13.31-1.18a1.18 1.18 0 11-2.36 0 1.18 1.18 0 012.36 0zm6.2 6.8c-1.63 0-2.91-.9-2.91-2.04 0-1.14 1.28-2.04 2.91-2.04 1.63 0 2.91.9 2.91 2.04 0 1.14-1.28 2.04-2.91 2.04zm0-3.08c-1.13 0-1.91.55-1.91 1.04s.78 1.04 1.91 1.04 1.91-.55 1.91-1.04-.78-1.04-1.91-1.04zm14.5-2.68c0-8.132-6.62-14.753-14.75-14.753-8.13 0-14.75 6.621-14.75 14.753S15.87 38.75 24 38.75c8.13 0 14.75-6.622 14.75-14.754zm-30.75 0c0-8.842 7.16-16.004 16-16.004s16 7.172 16 16.004C40 32.838 32.84 40 24 40S8 32.838 8 23.996z"
+        d="m28.086 3-2.267 2.809.972.785 2.268-2.809L28.086 3Zm-8.643.789.973-.785 2.266 2.81-.973.785-2.266-2.81ZM12.786 19.04l3.552-2.204.659 1.062-3.552 2.204-.66-1.063Zm17.47-1.14.66-1.062 3.552 2.205-.66 1.062-3.551-2.205Zm1.104 6.235a1.18 1.18 0 1 0 0-2.36 1.18 1.18 0 0 0 0 2.36Zm-13.31-1.18a1.18 1.18 0 1 1-2.36 0 1.18 1.18 0 0 1 2.36 0Zm6.2 6.8c-1.63 0-2.91-.9-2.91-2.04 0-1.14 1.28-2.04 2.91-2.04 1.63 0 2.91.9 2.91 2.04 0 1.14-1.28 2.04-2.91 2.04Zm0-3.08c-1.13 0-1.91.55-1.91 1.04s.78 1.04 1.91 1.04 1.91-.55 1.91-1.04-.78-1.04-1.91-1.04Zm14.5-2.68c0-8.132-6.62-14.753-14.75-14.753-8.13 0-14.75 6.621-14.75 14.753S15.87 38.75 24 38.75c8.13 0 14.75-6.622 14.75-14.754Zm-30.75 0c0-8.842 7.16-16.004 16-16.004s16 7.172 16 16.004C40 32.838 32.84 40 24 40S8 32.838 8 23.996Z"
```

Comment:
We are deferring to whatever `svgr`/`svgo` is outputting as best practice / optimized. For reference, the original svg is ~2kb:

```svg
<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
  <title>Anxious Emoji</title>
  <path fill-rule="evenodd" clip-rule="evenodd" d="M28.086 3L25.8186 5.80894L26.7912 6.59407L29.0586 3.78513L28.086 3ZM19.4433 3.78889L20.4164 3.00439L22.6821 5.81478L21.7089 6.59928L19.4433 3.78889ZM12.7859 19.0405L16.3376 16.8368L16.9966 17.8989L13.4449 20.1026L12.7859 19.0405ZM30.2568 17.9012L30.9162 16.8392L34.4675 19.0442L33.8082 20.1062L30.2568 17.9012ZM31.3599 24.1359C32.0116 24.1359 32.5399 23.6076 32.5399 22.9559C32.5399 22.3042 32.0116 21.7759 31.3599 21.7759C30.7082 21.7759 30.1799 22.3042 30.1799 22.9559C30.1799 23.6076 30.7082 24.1359 31.3599 24.1359ZM18.0499 22.9559C18.0499 23.6076 17.5216 24.1359 16.8699 24.1359C16.2183 24.1359 15.6899 23.6076 15.6899 22.9559C15.6899 22.3042 16.2183 21.7759 16.8699 21.7759C17.5216 21.7759 18.0499 22.3042 18.0499 22.9559ZM24.25 29.7568C22.62 29.7568 21.34 28.8568 21.34 27.7168C21.34 26.5768 22.62 25.6768 24.25 25.6768C25.88 25.6768 27.16 26.5768 27.16 27.7168C27.16 28.8568 25.88 29.7568 24.25 29.7568ZM24.25 26.6768C23.12 26.6768 22.34 27.2268 22.34 27.7168C22.34 28.2068 23.12 28.7568 24.25 28.7568C25.38 28.7568 26.16 28.2068 26.16 27.7168C26.16 27.2268 25.38 26.6768 24.25 26.6768ZM38.75 23.9962C38.75 15.8643 32.13 9.24269 24 9.24269C15.87 9.24269 9.25 15.8643 9.25 23.9962C9.25 32.1281 15.87 38.7497 24 38.7497C32.13 38.7497 38.75 32.1281 38.75 23.9962ZM8 23.9962C8 15.1541 15.16 7.9924 24 7.9924C32.84 7.9924 40 15.1641 40 23.9962C40 32.8383 32.84 40 24 40C15.16 40 8 32.8383 8 23.9962Z" fill="currentColor"/>
</svg>
```

The prev diff is 974kb:

```svg
<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
  <title>Anxious Emoji</title>
  <path fill-rule="evenodd" clip-rule="evenodd" d="M28.086 3l-2.267 2.809.972.785 2.268-2.809L28.086 3zm-8.643.789l.973-.785 2.266 2.81-.973.785-2.266-2.81zM12.786 19.04l3.552-2.204.659 1.062-3.552 2.204-.66-1.063zm17.47-1.14l.66-1.062 3.552 2.205-.66 1.062-3.551-2.205zm1.104 6.235a1.18 1.18 0 100-2.36 1.18 1.18 0 000 2.36zm-13.31-1.18a1.18 1.18 0 11-2.36 0 1.18 1.18 0 012.36 0zm6.2 6.8c-1.63 0-2.91-.9-2.91-2.04 0-1.14 1.28-2.04 2.91-2.04 1.63 0 2.91.9 2.91 2.04 0 1.14-1.28 2.04-2.91 2.04zm0-3.08c-1.13 0-1.91.55-1.91 1.04s.78 1.04 1.91 1.04 1.91-.55 1.91-1.04-.78-1.04-1.91-1.04zm14.5-2.68c0-8.132-6.62-14.753-14.75-14.753-8.13 0-14.75 6.621-14.75 14.753S15.87 38.75 24 38.75c8.13 0 14.75-6.622 14.75-14.754zm-30.75 0c0-8.842 7.16-16.004 16-16.004s16 7.172 16 16.004C40 32.838 32.84 40 24 40S8 32.838 8 23.996z" fill="currentColor"/>
</svg>
```

And the new one is 978kb:

```svg
<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
  <title>Anxious Emoji</title>
  <path fill-rule="evenodd" clip-rule="evenodd" d="m28.086 3-2.267 2.809.972.785 2.268-2.809L28.086 3Zm-8.643.789.973-.785 2.266 2.81-.973.785-2.266-2.81ZM12.786 19.04l3.552-2.204.659 1.062-3.552 2.204-.66-1.063Zm17.47-1.14.66-1.062 3.552 2.205-.66 1.062-3.551-2.205Zm1.104 6.235a1.18 1.18 0 1 0 0-2.36 1.18 1.18 0 0 0 0 2.36Zm-13.31-1.18a1.18 1.18 0 1 1-2.36 0 1.18 1.18 0 0 1 2.36 0Zm6.2 6.8c-1.63 0-2.91-.9-2.91-2.04 0-1.14 1.28-2.04 2.91-2.04 1.63 0 2.91.9 2.91 2.04 0 1.14-1.28 2.04-2.91 2.04Zm0-3.08c-1.13 0-1.91.55-1.91 1.04s.78 1.04 1.91 1.04 1.91-.55 1.91-1.04-.78-1.04-1.91-1.04Zm14.5-2.68c0-8.132-6.62-14.753-14.75-14.753-8.13 0-14.75 6.621-14.75 14.753S15.87 38.75 24 38.75c8.13 0 14.75-6.622 14.75-14.754Zm-30.75 0c0-8.842 7.16-16.004 16-16.004s16 7.172 16 16.004C40 32.838 32.84 40 24 40S8 32.838 8 23.996Z" fill="currentColor"/>
</svg>
```

Reply:
They're identical to the eye (Github won't let me attach them), and I imagine the same applies to the majority of this diff.